.github/workflows/build.yml

```diff
@@ -39,14 +56,9 @@ jobs:
           --health-retries 5
         ports:
           - 5432:5432
-
     strategy:
-      fail-fast: false
```

Comment:
don't we want this? I think it can save some time and I think generally it's more valuable than to wait for everything

Reply:
Hm I don't remember removing this line ....